Atlas-Security.md

```diff
@@ -0,0 +1,89 @@
+# ATLAS Security
+
+The following sections discuss how WebAPI security is utilized in [ATLAS](https://github.com/OHDSI/Atlas). 
+
+## Configuring ATLAS
+
+Now that we have the OHDSI WebAPI configured, we can now setup ATLAS to expect a secure OHDSI WebAPI. 
+
+Placing a `config-local.js` file inside the root `atlas/js` file in your web installation will allow you to override the configuration settings without requiring changes to the Github repository or accidentally pushing your local information to Github.  The following code configures ATLAS to expect a secure OHDSI WebAPI installation and configures it to use our newly created demonstration database.
+
+```javascript
+define([], function () {
+	var configLocal = {};
+
+	configLocal.userAuthenticationEnabled = true;
+
+	configLocal.api = {
+		name: 'Demo Environment',
+		url: 'http://localhost:8080/WebAPI/'
+	};
+
+	configLocal.authProviders = [{
+		"name": "Local Security Test DB",
+		"url": "user/login/db",
+		"ajax": true,
+		"icon": "fa fa-database",
+		"isUseCredentialsForm": true
+	}];
+
+	return configLocal;
+});
+```
+
+In the example above, the `configLocal.userAuthenticationEnabled` setting is updated to **true** to enable security in [ATLAS](https://github.com/OHDSI/Atlas). The `configLocal.api` contains the name and URL to connect to our WebAPI instance. `configLocal.authProviders` is an array that enables the specification of 1 or more authentication providers. A full list of authentication providers is included in the ATLAS `/js/config/app.js` `appConfig.authProviders` section and the ones that are enabled in your environment should be included in the `configLocal.authProviders` list.
+
+For those that followed the [[Basic Security Configuration]], the settings above will work. For those using OAuth/AD, please use the relevant configuration for your setup.
+
+## Defining an Administrator
+You should now be able to load ATLAS and find that you can login to the environment using the authentication provider configured in the previous step.  However, you will have limited permissions.  The following query will list the current permissions that your login has in the database:
+
+```sql
+select sec_user.id as user_id, login, sec_role.id as role_id, sec_role.name as role_name
+from sec_user
+join sec_user_role on sec_user.id = sec_user_role.user_id
+join sec_role on sec_user_role.role_id = sec_role.id
+```
+
+To grant yourself administrator privileges you can run the following query:
+
+```sql
+insert into sec_user_role (user_id, role_id) values (1000,2)
```

Comment:
the value of '1000' here I think depends on what the user_id is of the actual user. Should we indicate that?